grpc-gateway/client/cloudCodec.go

```diff
@@ -0,0 +1,66 @@
+package client
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	"github.com/plgd-dev/go-coap/v2/message"
+	"github.com/plgd-dev/kit/codec/cbor"
+	"github.com/plgd-dev/kit/codec/json"
+)
+
+// CloudCodec encodes in application/vnd.ocf+cbor and decodes json/coap/text.
+type CloudCodec struct{}
```

Comment:
Why CloudCodec ? Name is not very descriptive. StandardMessageCodec ? OCFCodec? MessageCodec?